### ISSUE TITLE
Too many folders created

### ISSUE THREAD

**Issue by Crimsoonhack** [sim0n00ps/OF-DL#296]
Is there a way not to create so many folders when downloading content? Currently, folders are created for paid content, for free content, for paid messages, for free messages.... even inside the `__user_data__` folder there are two more folders up to the downloaded profiles.

Is it possible to reduce so many folders and unify it a bit? That is, at least dispense with the paid and free folders.

**Comment by lucacham** [comment]
I agree with this. Every time I open the destination folder, I need to open too many folders

**Comment by sim0n00ps** [comment]
Ok, what structure would you guys prefer, if you can give examples then I'll see what I can do

**Comment by Crimsoonhack** [comment]
> Ok, what structure would you guys prefer, if you can give examples then I'll see what I can do

For example, it would be nice if it were like this:

`profiles > username > [photos] [videos] [messages]` 

Inside `[messages]` all together, videos and photos

**Comment by fidelio959** [comment]
I also think this is a good solution:
profiles > username > [photos] [videos] [messages]
Thanks

**Comment by BluesFanUK** [comment]
Agree, the metadata and avatars/headers could sit in the main username folder, if the avatar was renamed to 'folder' that would also create a folder preview in Windows. (less important).  

Just realised I can use the rawText for the file names too, just keeps getting better.👌

**Comment by SkylineRcr** [comment]
Did this ever get added?
